### ISSUE TITLE
4.4.0 - Always Shows Cleaners Hidden When Starting

### ISSUE THREAD

**Issue by 0pLuS0** [bleachbit/bleachbit#1299]
**Describe the bug**
Hide irrelevant cleaners needs to be constantly checked and unchecked to show all the options everytime bleachbit is started. This happens both in Windows 10 and Linux.

**To Reproduce**
1. start bleachbit
2. Main UI shows only few options as if 'Hide irrelevant cleaners' is checked, when it's not.
3. Need to check, then uncheck 'Hide irrelevant cleaners' to show all the options.

The error happens (sometimes/every time).

Every time!

**Expected behavior**
'Hide irrelevant cleaners'  is not checked, so all the options should always be in view when starting bleachbit..

**Screenshots**
![4 4 0](https://user-images.githubusercontent.com/36190223/140869988-f7e2a90f-2ef9-452e-ad56-72c9251ead40.jpg)

**Desktop (please complete the following information):**
 - OS: Windows 10 and Slackware 14.2
 - BleachBit version 4.4.0

**Comment by abitrolly** [comment]
I believe that's fixed in https://www.bleachbit.org/news/bleachbit-441-beta

**Comment by 0pLuS0** [comment]
Is 4.4.1 the next release and if so, when will it be released as stable?

Thanks

**Comment by az0** [comment]
> Is 4.4.1 the next release and if so, when will it be released as stable?

Yes, 4.4.1 beta is the next release after 4.4.0 stable. The release after 4.4.1 beta will be 4.4.2 stable, and assuming there are no show stoppers, then 4.4.2 could be very soon.

Did 4.4.1 fix the "always shows cleaners hidden when starting" issue for you?

**Comment by 0pLuS0** [comment]
My bad, I saw that later... 

Thanks

**Comment by abitrolly** [comment]
@0pLuS0 have you got a chance to try 4.4.1 beta?

**Comment by 0pLuS0** [comment]
No worries, I'll wait for 4.4.2

Thanks

**Comment by az0** [comment]
> No worries, I'll wait for 4.4.2

The wait is over: [BleachBit 4.4.2 final](https://www.bleachbit.org/news/bleachbit-442)
